helpers/deploymentConfig.ts

```diff
@@ -284,7 +298,11 @@ export const redstoneFeed: Config = {
     XVS: "0xd9a66Ff1D660aD943F48e9c606D09eA672f312E8",
   },
   zksyncsepolia: {
-    XVS: "0x3AeCac43A2ebe5D8184e650403bf9F656F9D1cfA",
+    XVS: "",
```

Comment:
I would remove the entry for `zksyncsepolia`

Reply:
[affc0c8](https://github.com/VenusProtocol/oracle/pull/215/commits/affc0c89dff351b19422b13c15e78f8a9c37096d)